ChangeLog

```diff
@@ -1,3 +1,7 @@
+* Add QUIRK_MFM_CLOCK.  This quirk was needed to read some old UDOS
+  1526 V.3 or V.2 disks, which have clock bits set in certain places
```

Comment:
Any reason why the "V"'s numbers are in reverse ascending order?  Both here in the commit log you have "V.3 or V.2" instead of "V.2 or V.3".  Unless there's a reason, it runs afoul of the element of least surprise.

Otherwise, all the changes look straightforward and good.  You even remembered to change the parselog which I would have forgotten about without your delta.  :)